src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '../index.css';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] {OPTIONAL] Let's not overuse eslint-disable, if your component doesn't handle any state if should be a functional component (like App), not a class one.

[Check this article about stateful and stateless components for more on that .](https://programmingwithmosh.com/javascript/stateful-stateless-components-react/)